README.md

```diff
@@ -78,25 +54,56 @@ Do this once:
 mv src/third_party/libreofficekit{,.old}
 ```
 
-Linux/MacOs
+**Linux/macOS**
 ```bash
 # link src/third_party/libreofficekit to your local libreofficekit/libreoffice-core
 ln -s ../libreofficekit/libreoffice-core src/third_party/libreofficekit
 ```
 
-Windows
+**Windows**
 ```bash
 # note you must use an administrator command prompt, and the source and target are flipped
 mklink /D ".\src\third_party\libreofficekit" "..\libreofficekit\libreoffice-core"
 ```
 
+When you make a new local build of `libreofficekit`, make a new Electron build with `UPDATE_LOK=true`:
+```shell
+UPDATE_LOK=true scripts/e build
+```
+
+## Using a debug build in another project
+```shell
+# copy zipped runtime files to ~/another_project
+INJECT_TO="~/another_project" scripts/e build
+
+# this copies a fresh build of dist.zip, mksnapshot.zip, mksnapshot_args, and gen/v8/electron.S
+# dist.zip can be expanded into node_modules/electron/dist
+# mksnapshot.zip can be expanded into node_modules/electron-mksnapshot/bin
+# all other files can be copied into node_modules/electron-mksnapshot/bin
```

Comment:
are these meant to be the same folder name?

Reply:
No, these are two different tools: electron and electron-mksnapshot

Reply:
Right, just double checking the folders were correct cause sometimes I copy and paste without changing the name